spyderplugins/widgets/pylintgui.py

```diff
@@ -61,7 +61,8 @@ def get_pylint_version():
                                shell=True if os.name == 'nt' else False)
     lines = to_unicode_from_fs(process.stdout.read()).splitlines()
     if lines:
-        match = re.match('(pylint3?|pylint-script.py) ([0-9\.]*)', lines[0])
+        regex = '({0}|pylint-script.py) ([0-9\.]*)'.format(PYLINT)

```

Comment:
Sorry, this was only necessary to detect the Pylint version, so I think your fix is the right one :-)